dss/util/email.py

```diff
@@ -0,0 +1,76 @@
+import boto3
+from botocore.exceptions import ClientError
+
+# The character encoding for the email.
+CHARSET = "UTF-8"
+
+# Create a new SES resource and specify a region.
+client = boto3.client('ses')
+
+def send_email(sender: str, to: str, subject: str, html: str, text: str) -> str:
+    try:
+        # Provide the contents of the email.
+        response = client.send_email(
+            Destination={
+                'ToAddresses': [
+                    to
+                ],
+            },
+            Message={
+                'Body': {
+                    'Html': {
+                        'Charset': CHARSET,
+                        'Data': html,
+                    },
+                    'Text': {
+                        'Charset': CHARSET,
+                        'Data': text,
+                    },
+                },
+                'Subject': {
+                    'Charset': CHARSET,
+                    'Data': subject,
+                },
+            },
+            Source=sender,
+        )
+    # Display an error if something goes wrong.
+    except ClientError as e:
+        return e.response['Error']['Message']
```

Comment:
You're not going to raise an exception here? What's the justification for that?

Reply:
good catch, that was done to debug SES sending issues. 
Fixed now.